### ISSUE TITLE
[Accessibility]A11y_VSCode_KeyboardShortcuts_Keyboard: Trigger Suggest (Ctrl+Alt+Space) shortcut command has some another functionality.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52677]
Environment Details:
VSCode Version : 1.24.0

Additional Details:
MAS Violated: MAS2.1.1

Repro Steps:

Launch VS Code.
Open Keyboard Shortcuts.

Actual:
Trigger Suggest (Ctrl+Alt+Space) shortcut command has some another functionality. All the commands related to Trigger suggest has same functionality,only this command has some another functionality.

Expected:
All the commands related to Trigger suggest should have same behavior.

Recommendations:
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

MAS Reference
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

Attachment for reference:
[A11y VSCode KeyboardShortcuts Trigger Suggest.pptx](https://github.com/Microsoft/vscode/files/2127539/A11y.VSCode.KeyboardShortcuts.Trigger.Suggest.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by chrmarti** [comment]
Could be #52681. Note the additional bindings shown for each command.

**Comment by isidorn** [comment]
The when clause is different. This is all good
